dot/rpc/modules/state_test.go

```diff
@@ -1129,3 +1129,147 @@ func TestStateModuleQueryStorage(t *testing.T) {
 		})
 	}
 }
+
+func TestStateModuleQueryStorageAt(t *testing.T) {
+	t.Parallel()
+	errTest := errors.New("test error")
+
+	type fields struct {
+		storageAPIBuilder func(ctrl *gomock.Controller) StorageAPI
+		blockAPIBuilder   func(ctrl *gomock.Controller) BlockAPI
+	}
+	type args struct {
+		in0 *http.Request
```

Comment:
Since `in0` is always ignored, just remove it and always pass nil in the subtest code?